### ISSUE TITLE
We need a new ball model

### ISSUE THREAD

**Issue by tuxbotix** [HULKs/hulk#337]
Current ball model is quite 'jumpy' and does not accurately predict ball state.

## Goal
Ddvelop a new ball model which can accurately predict future state of the ball and be easily tunable for different fields.

## Requirements 
1. The motion state should be automatically determined or be inferred as part of a black box style modellng.
    - rolling of the ball during slow kicks or when slowing down
    - brief flying or non rolling, fast movements (I.e. fast kicks)
2. The above physical states determined without explicit input
    - i.e. from ball detection whether it is rolling or otherwise.
3. Ideally it should be self correcting (feedback loop) so that a close enough initial tuning will be sufficient.
4. The model should be easy to tune up, 10+ parameters are too much!
    - or at least have an easy to replicate and portable automatic method
5. Design a repeatable experiment for development , evaluation and tuning if the model later at events.

**Comment by PasGl** [comment]
Currently Team-messages only contain a position for the ball.
It might be worth a thought to add more information about the ball state to these messages, so that Players that don't see the ball themselves still can make more elaborate time-to-reach-ball calculations.

**Comment by tuxbotix** [comment]
Yes it makes sense to add these into the team messages, we put that under the time to reach ball improvements.
https://github.com/HULKs/hulk/issues/297

**Comment by schluis** [comment]
Please check the current status and whether it is still applicable.

**Comment by schmidma** [comment]
We should develop a completely new 3D approach to ball tracking with multiple tracks... Closing for now
